<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="mBeginTag" stdset="0">
   <string/>
  </property>
  <property name="mEndTag" stdset="0">
   <string/>
  </property>
  <layout class="QGridLayout" name="mLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="mSettingsTable">
     <column>
      <property name="text">
       <string>Add?</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Command</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
